runtimes/runtimes/lsp/router/loggingServer.ts

```diff
@@ -0,0 +1,44 @@
+import { InitializeParams, InitializeResult, Logging } from '../../../server-interface'
+import { LspServer } from './lspServer'
+import { Connection } from 'vscode-languageserver/node'
+import { Encoding } from '../../encoding'
+import { DEFAULT_LOG_LEVEL, isValidLogLevel, LoggingImplementation, LogLevel } from '../../util/loggingUtil'
+
+export class LoggingServer {
+    private logger: Logging
+    private lspServer: LspServer
+
+    constructor(
+        private lspConnection: Connection,
+        private encoding: Encoding
+    ) {
+        this.logger = new LoggingImplementation(DEFAULT_LOG_LEVEL as LogLevel, this.lspConnection)
+        this.lspServer = new LspServer(this.lspConnection, this.encoding, this.logger)
+        this.lspServer.setInitializeHandler(async (params: InitializeParams): Promise<InitializeResult> => {
+            this.updateLoggingLevel(params.initializationOptions?.logLevel ?? ('log' as LogLevel))
+            return {
+                capabilities: {},
+            }
+        })
+        this.lspServer.setDidChangeConfigurationHandler(async params => {
+            const logLevelConfig = await lspConnection.workspace.getConfiguration({
+                section: 'aws.logLevel',
```

Comment:
We need to add this LSP configuration details and supported values to documentation, to communicate it to destinations https://github.com/aws/language-server-runtimes/blob/main/runtimes/README.md#logging